index.js

```diff
@@ -1 +1,682 @@
-console.log("hello world")
\ No newline at end of file
+ let toggle=document.querySelector('.toggle')
+let header=document.querySelector('header')
+let dropdown=document.querySelector('.drop-down ')
+let socials=document.querySelector('.socials-ul')
+let secondRow=document.querySelector('.second-roww ');
+let secondRoww=document.querySelector('.second-row ');
+ toggle.addEventListener('click',translate)
+ function translate(){
+     dropdown.classList.toggle('nav-active');
+     secondRow.classList.toggle('nav-active');
+     socials.classList.toggle('nav-active');
+     header.classList.toggle('header');
+     if(secondRow.classList.contains('nav-active')){
+        //  console.log('hey')
+     }
+     else{
+        //  console.log('hi')
+        secondRow.classList.add('second-roww')
+        secondRow.classList.remove('second-row')
+     }
+     if(secondRow.classList.contains('nav-active')){
+         
+        secondRow.classList.remove('second-roww')
+       secondRow.classList.add('second-row')
+    }
+    else{
+        // console.log('hi')
+       secondRow.classList.add('second-roww')
+       secondRow.classList.remove('second-row')
+    }
+ }
+ 
+
+//  courosel
+var slideIndex = 1;
+showSlides(slideIndex);
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+// Thumbnail image controls
+function currentSlide(n) {
+  showSlides(slideIndex = n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var dots = document.getElementsByClassName("dot");
+  if (n > slides.length) {slideIndex = 1}
+  if (n < 1) {slideIndex = slides.length}
+  for (i = 0; i < slides.length; i++) {
+      slides[i].style.display = "none";
+  }
+  for (i = 0; i < dots.length; i++) {
+      dots[i].className = dots[i].className.replace(" active", "");
+  }
+  slides[slideIndex-1].style.display = "block";
+  dots[slideIndex-1].className += " active";
+}
+var slideIndex = 0;
+showSlides();
+
+function showSlides() {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  for (i = 0; i < slides.length; i++) {
+    slides[i].style.display = "none";
+  }
+  slideIndex++;
+  if (slideIndex > slides.length) {slideIndex = 1}
+  slides[slideIndex-1].style.display = "block";
+  setTimeout(showSlides, 5000); // Change image every 2 seconds
+}
+
+// window.addEventListener('click',   translatee)
+ 
+  // document.querySelector('body').style.backgroundColor='white'
+  
+   
+
+ 
+//objects for the cart items
+const computers=[
```

Comment:
extract this variable into an exported module and import it into to break up this file that is very very long. 

create issue

Reply:
rodger ....